airflow/utils/platform.py

```diff
@@ -63,6 +65,7 @@ def get_airflow_git_version():
     return git_version
 
 
+@cache
```

Comment:
I added this as there is 0 need to call this 20k times when creating a dag -- it can't change at runtime.

(Okay, it _technically_ could change the name at runtime, but that is very much an edge case, and not to mention that the value created in the scheduler doesn't matter, it will be re-overwritten when the Task executes, and that is the interesting value anyway

Reply:
Scratch that -- I decided to remove the setting of `unixname` attribute for TaskInstances when they are created -- it doesn't make sense to have a value there until they have been executed onec.

Reply:
Aaand reverted  the "remove setting unixname" as the API schema _requires_ a value for unixname, so changing that would be a bigger/separate change.